xbmc/utils/Locale.cpp

```diff
@@ -233,3 +252,30 @@ void CLocale::Initialize()
     StringUtils::ToUpper(m_territory);
   }
 }
+
+int CLocale::GetMatchCount(const std::string& locale) const
```

Comment:
It's quite early at the morning  so I can be wrong ;) I guess you calculate a score or rank here. If so I would rename the method.